integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -324,6 +324,17 @@ <h3 class="heading font-bold text-black">
                                         {% trans 'Archive this page' %}
                                     </button>
                                 {% endif %}
+                                {% if perms.cms.change_page and not page.archived %}
+                                    <label>
+                                        {% trans 'Refresh date' %}
+                                    </label>
+                                    <a href="{% url 'refresh' page_id=page_form.instance.id region_slug=request.region.slug language_slug=language.slug  %}"
```

Comment:
Since this view changes the database, we need to convert it into a form + button instead of normal link.